src/catch2/catch_matchers_templates.hpp

```diff
@@ -0,0 +1,269 @@
+#ifndef TWOBLUECUBES_CATCH_MATCHERS_TEMPLATES_HPP_INCLUDED
+#define TWOBLUECUBES_CATCH_MATCHERS_TEMPLATES_HPP_INCLUDED
+
+#include <catch2/catch_common.h>
+#include <catch2/catch_matchers.h>
+#include <catch2/catch_stringref.h>
+
+#include <array>
+#include <algorithm>
+#include <memory>
+#include <string>
+#include <type_traits>
+#include <utility>
+#include <vector>
```

Comment:
This include should not be needed (I am also not sure what <memory> is used for, but I did not look hard)

Reply:
I forgot to remove the <vector> include.